.env.example

```diff
@@ -0,0 +1,11 @@
+# Optional
+GIRAFFE_ENDPOINT=https://graphql.dev.hedvigit.com/graphql
+
+# Optional
+GIRAFFE_WS_ENDPOINT=wss://graphql.dev.hedvigit.com/subscriptions
+
+# Optional
+CONTENT_SERVICE_ENDPOINT=https://graphql.dev.hedvigit.com
```

Comment:
In H.OPE. we do not point towards our own dev cluster in these `.example` files, rather something like:
```suggestion
# Optional
GIRAFFE_ENDPOINT=<giraffe-endpoint>

# Optional
GIRAFFE_WS_ENDPOINT=<giraffe-ws-endpoint>

# Optional
CONTENT_SERVICE_ENDPOINT=<giraffe-service-endpoint>
```
Could we worth considering